dexbot/test.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python3
+
+from bitshares.bitshares import BitShares
+import unittest
+import time
+import threading
+import logging
+from dexbot.bot import BotInfrastructure
+
+
+logging.basicConfig(
+        level=logging.INFO,
+        format='%(asctime)s %(levelname)s %(message)s'
+    )
+
+
+TEST_CONFIG={
+    'node':'wss://node.testnet.bitshares.eu',
+    'bots':{
+        'echo':
+        {
+            'account':'aud.bot.test4',
+            'market':'TESTUSD:TEST',
+            'module':'dexbot.strategies.echo'
+        },
+        'follow_orders':
+    {
+            'account':'aud.bot.test4',
+            'market':'TESTUSD:TEST',
+            'module':'dexbot.strategies.follow_orders',
+            'spread':5,
+            'reset':True,
+            'staggers':2,
+            'wall':5,
+            'staggerspread':5,
+            'min':0,
+            'max':100000,
+            'start':50
+        }}}
+
+KEYS=['5JV32w3BgPgHV1VoELuDQxvt1gdfuXHo2Rm8TrEn6SQwSsLjnH8']
+
+
+
+class TestDexbot(unittest.TestCase):
+
+    def test_dexbot(self):
+        btsi = BitShares(node=TEST_CONFIG['node'], keys=KEYS)
```

Comment:
In python it's more common place to use descriptive variable names, if you could change the variable names that would be sweet. for example btsi -> bitshares_instance